bugsnag-spring/src/main/java/com/bugsnag/BugsnagSpringConfiguration.java

```diff
@@ -64,9 +64,8 @@ ScheduledTaskBeanLocator scheduledTaskBeanLocator() {
      * If using Logback, stop any configured appender from creating Bugsnag reports for Spring log
      * messages as they effectively duplicate error reports for unhandled exceptions.
      */
-    @PostConstruct
-    @SuppressWarnings("checkstyle:emptycatchblock")
-    void excludeLoggers() {
+    @Override
+    public void afterPropertiesSet() throws java.lang.Exception {
```

Comment:
Any specific reason the FQN is being used here?
```suggestion
    public void afterPropertiesSet() throws Exception {
```